RecoPixelVertexing/PixelTriplets/interface/CACut.h

```diff
@@ -11,6 +11,7 @@
 #define RecoPixelVertexing_PixelTriplets_src_CACut_h
```

Comment:
```suggestion
#define RecoPixelVertexing_PixelTriplets_CACut_h
```
perhaps the same update can be done in RecoPixelVertexing/PixelTriplets/interface/CAGraph.h as well

Reply:
done